packages/@react-spectrum/tooltip/src/TooltipTrigger.tsx

```diff
@@ -50,7 +52,8 @@ function TooltipTrigger(props: SpectrumTooltipTriggerProps) {
     crossOffset,
     isOpen: state.isOpen,
     shouldFlip: props.shouldFlip,
-    containerPadding: props.containerPadding
+    containerPadding: props.containerPadding,
+    arrowCrossSize: 8 + arrowTotalCrossPadding
```

Comment:
is this 2x? or why does it have a const 8 with a variable that is also 8?

Reply:
8 here stands for the arrow width and arrowTotalCrossPadding is invisible space so visible part of the arrow doesn't overlap the part of overlay with rounder corner. I'm updating to get the arrow size dynamically now